README.md

```diff
@@ -202,6 +209,8 @@ File: .patch file to use for the livepatch package build.
 Options:
   --arch       CPU architecture for livepatch. arm64 and x86_64 are supported.
                Default is x86_64.
+  -c, --callbacks   .c file implementing callbacks for livepatch
```

Comment:
We should reformat this message so that the command line descriptions are all aligned.

Reply:
Done in README. Different commit can handle the output of llpatch.